Wabbajack/AppState.cs

```diff
@@ -165,8 +183,57 @@ public ICommand VisitNexusSiteCommand
             }
         }
 
-        public string _nexusSiteURL = null;
+        public ICommand OpenModListPropertiesCommand
+        {
+            get
+            {
+                return new LambdaCommand(() => true, () => OpenModListProperties());
+            }
+        }
```

Comment:
Can scrunch this a bit to `public ICommand OpenModListPropertiesCommand => new LambdaCommand(() => true, OpenModListProperties);`

Additionally, maybe if you add a ctor to LambdaCommand that doesn't have a canExecute parameter (eg, internally sets it to always be executable), this could just be
`public ICommand OpenModListPropertiesCommand => new LambdaCommand(OpenModListProperties);`